src/utils/common.ts

```diff
@@ -478,3 +478,13 @@ export function isCoin(assetContractId: AssetContractId): boolean {
       .coinContractId === assetContractId,
   );
 }
+/**
+* Clean options from members that cause issues when sending messages
+* or when signing transactions.
+*/
```

Comment:
Missing single spaces before the asterisks.

Consider installing this VS Code extension: https://marketplace.visualstudio.com/items?itemName=aaron-bond.better-comments

Reply:
It lives in my VS Code rent free! I just moved this code from another file and didn't notice the linting error